src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/index.jsx

```diff
@@ -316,14 +316,21 @@ class GlobalSelectionHeader extends React.Component {
       return;
     }
 
+    const {organization, params} = this.props;
     if (forceProject) {
       // this takes precendence over the other options
       newProject = [getProjectIdFromProject(forceProject)];
     } else if (requestedProjects && requestedProjects.length > 0) {
       // If there is a list of projects from URL params, select first project from that list
       newProject = [requestedProjects[0]];
-    } else {
-      // Otherwise, get first project from org that the user is a member of
+    } else if (
+      organization &&
+      params &&
+      organization.slug &&
+      organization.slug === params.orgId
```

Comment:
Can you use [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) for this?

Reply:
Done. See https://github.com/getsentry/sentry/pull/16433